### DISCUSSION TITLE
Configurable default filters for projects

### DISCUSSION THREAD

**Discussion by AlexVanderbist** [spatie/flareapp.io-roadmap#84] [Ideas]
The current default filter for errors in a project is `is:unresolved`. Previously, it was `is:unresolved is:unsnoozed`. It would be helpful to have the option to customize this default filter for each project.

**Comment by ChrisWorkhouse** [comment]
It would also be nice to have a selection of common (customisable ideally - not sure if this is best Team or User level) filters that you can quickly click and more easily switch between

For example, quick switching between [Unresolved is:unresolved] [Resolved is:resolved] [Snoozed is:snoozed]

**Comment by breadthe** [comment]
+1 to this. I think `is:unresolved is:unsnoozed` should probably be the default.

I think there should be a default project-level filter that is always applied. Now, if I click the project URL it clears the `is:unsnoozed`. Even better, this should be a user-level filter as well.
